src/Debugging-Core/Context.extension.st

```diff
@@ -245,38 +245,46 @@ Context class >> simulatePrimitiveNumber: num with: simulator [
 Context >> stepToHome: aContext [
```

Comment:
This is strange, why is this method an extension of the debugger?

Reply:
Some time ago we move it out of Context to make a smaller bootstraped image.
This code is only used by the debugger, it is not used by the exception handling mechanism.